cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/heat/OpenStackHeatNetworkConnector.java

```diff
@@ -52,6 +52,11 @@ public void createProviderSpecificNetworkResources(NetworkResourcesCreationReque
         throw new OpenStackResourceException("CDP does not support Openstack.");
     }
 
+    @Override
+    public SubnetSelectionResult chooseSubnetsForPrivateEndpoint(Collection<CloudSubnet> subnetMetas, boolean existingNetwork) {
+        throw new OpenStackResourceException("CDP does not support Openstack.");
```

Comment:
we have UnsupportedOperationException, too

Reply:
Yes, but since other methods do use that exception, I do not want to change the others.